ts/main.ts

```diff
@@ -22,40 +53,117 @@ if (!$imgUrlInput) throw new Error('$imgUrlInput query failed');
 if (!$titleInput) throw new Error('$titleInput query failed');
 if (!$notesInput) throw new Error('$notesInput query failed');
 
-// add event listener for img url input
-$imgUrlInput.addEventListener('input', () => {
-  const $newImgUrl = $imgUrlInput.value;
+// add event listeners for DOM
+document.addEventListener('DOMContentLoaded', () => {
+  // Show the view that was previously active before the page refresh
+  const previousView = data.view || 'entries';
+  viewSwap(previousView);
+  toggleNoEntries();
 
-  // set the src attribute
-  $imgPreview.src = $newImgUrl || './images/placeholder-image-square.jpg';
-  $imgPreview.alt =
-    'updated preview image' || 'default placeholder image for journal entry';
+  // add event listener for the "New" button to show the entry form
+  const newEntryButton = document.getElementById('view-new-entry-link')!;
+  newEntryButton.addEventListener('click', (event) => {
+    event.preventDefault(); // Prevent the default action (link behavior)
+    viewSwap('entry-form'); // Switch to the new entry form view
+  });
```

Comment:
This shouldn't be inside of this event. Also `event.preventDefault` doesn't have an impact here.